src/styles/settings.scss

```diff
@@ -11,3 +11,36 @@ $primary: #d58e32;
 $text-color: #2a2a2a;
 
 $form-border-color: #292929;
+
+/* CUSTOMIZE HOVER */
+    /* HOVER PARAMS IN TOPBAR */
+    $TB-transition-color: 250ms;
```

Comment:
Are you sure that we need all of these variables? I mean there are a lot of duplications so maybe we can merge it to one variable